src/components/FeesField/EthereumKind.js

```diff
@@ -34,7 +34,7 @@ class FeesField extends Component<Props & { fees?: Fees, error?: Error }, *> {
     return (
       <GenericContainer error={error} help="Gas">
         <InputCurrency
-          defaultUnit={units[1]}
+          defaultUnit={units.length > 1 ? units[1] : units[0]}
```

Comment:
Should we consider again to have `defaultUnit` on currency ?

Reply:
here it's not really the defaultUnit I want to pick, `unit[0]` is still the default prop.
this one is more the default gas unit to use for Ethereum. not sure it worth introducing this concept.. I think in future, we might actually want to pick one dynamically, like probably the smallest unit where the default value is > 1 ?

for instance, for bitcoin fee per byte, we'll use satoshis, but imagine if btc price is very very high, maybe you know what to use mBTC, etc.. anyway. for now it's ok to hardcode it (for btc, we always take the `last()`.  ETH have way too much magnitude to use `last()` :D